lib/generate-report.js

```diff
@@ -58,6 +58,8 @@ function generateReport(options) {
   const saveCollectedJSON = options.saveCollectedJSON;
   const displayDuration = options.displayDuration || false;
   const durationInMS = options.durationInMS || false;
+  const pageTitle = options.pageTitle || 'Multiple Cucumber HTML Reporter';
+  const pageFooter = options.pageFooter || "<div class='created-by'><p>Created by wswebcreation. Find me on:</p><a href='http://www.wswebcreation.nl/' target='_blank'><i class='fa fa-rss-square fa-2x'></i></a><a href='https://github.com/wswebcreation/' target='_blank'><i class='fa fa-github-square fa-2x'></i></a><a href='http://nl.linkedin.com/in/wimselles' target='_blank'><i class='fa fa-linkedin-square fa-2x'></i></a><a href='http://stackoverflow.com/users/5911978/wswebcreation' target='_blank'><i class='fa fa-stack-overflow fa-2x'></i></a></div>";
```

Comment:
I'de like the html in the file where it belongs, so please check in the template files what needs to be loaded, if / else can be used there

Reply:
pageTitle - is which shows as HTML file title ( I like to have specific project name instead)
pageFooter - again same as above comment

what i see, currently don't have support to load the custom template. So i though this will be quicker to acheive